gatsby/src/components/looking-for-something/looking-for-something.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import classes from './looking-for-something.module.scss';
+import { ArrowForwardIos, Phone } from '@material-ui/icons';
+
+const defaultValues = {
+  searchingHeader: (
+    <>
+      <strong>Stále</strong> hledáte?
+    </>
+  ),
+  searchingDescription:
+    'Nenašli jste odpověď v žádné kategorii, tématu ani častých otázkách?',
+  callHeader: (
+    <>
+      Volejte{' '}
+      <strong>
+        <a className={classes.phoneNumber} href="tel:1221">
+          1221
+        </a>
+      </strong>
+    </>
+  ),
+  callDescription: 'Pro urychlení si prosím připravte svůj konkrétní dotaz.',
+};
+
+type Props = typeof defaultValues;
+
+const LookingForSomething = ({
+  searchingHeader,
+  searchingDescription,
+  callHeader,
+  callDescription,
+}: Props = defaultValues) => {
+  return (
+    <div className={classes.container}>
+      <section className={classes.section}>
+        <header className={classes.header}>{searchingHeader}</header>
+        <ArrowForwardIos />
+        <p className={classes.description}>{searchingDescription}</p>
+      </section>
+
+      <hr className={classes.separator} />
```

Comment:
I think it could be covered via `border-bottom` sorry did not mention it in the beginnig